storage/spider/mysql-test/spider/t/create_temporary_table.test

```diff
@@ -0,0 +1,29 @@
+# This test tests for table creation option
+source include/have_log_bin.inc;
+--disable_warnings
+--disable_query_log
+--disable_result_log
+--source test_init.inc
+--source slave_test_init.inc
+--enable_result_log
+--enable_query_log
+
+--echo drop and create databases
+--connection master_1
+DROP DATABASE IF EXISTS auto_test_local;
+CREATE DATABASE auto_test_local;
+USE auto_test_local;
+
+--echo create temporary table
+DROP TABLE IF EXISTS tb_l;
+echo CREATE TEMPORARY TABLE tb_l (
+  a INT
+) ENGINE=Spider;
+--error ER_ILLEGAL_HA_CREATE_OPTION
+eval CREATE TEMPORARY TABLE tb_l (
+  a INT
+) ENGINE=Spider;
+
+--echo drop databases
+DROP DATABASE IF EXISTS auto_test_local;
+
```

Comment:
Don't need an `echo` for cleanup. Given the table shouldn't be created, not even a `DROP TABLE` is necessary.

Reply:
Convention is now to end each test with a marker of the version to aid in mering upwards where tests are added. So end with:

```

--echo #
--echo # End of 10.2 tests
--echo #
```